core/templates/pages/privacy-page/privacy-page.component.html

```diff
@@ -409,3 +409,9 @@ <h2>Contacting Us</h2>
     <em>Last updated: 24 May 2018</em>
   </div>
 </div>
+
+<style>
```

Comment:
Could you please take a look at how per-page CSS is handled in other pages/components? I think normally we add a prefix to the rule so that the rule doesn't get applied outside the page/component.

Reply:
It's handled the same way, by adding the `<style>` HTML element, such as in the terms page, and blog dashboard page.

In this case I'm adding the style to all elements  `a`, since the goal is to improve a11y by fullfiling [WCAG SC 1.4.1](https://www.w3.org/WAI/WCAG22/Understanding/use-of-color) in these specific pages.

Angular automatically restricts the new style tag added to elements inside that div ID (added by Angular), so it should not propagate. Just tested it comparing to the dev version. Here is how the computed style looks with my fix.
<img width="414" alt="Screenshot 2024-11-17 at 18 24 14" src="https://github.com/user-attachments/assets/5c61aa64-f5e8-46f7-8f39-8b9ff1715289">

Here is the version in dev:

<img width="443" alt="Screenshot 2024-11-17 at 18 32 45" src="https://github.com/user-attachments/assets/d1fa1735-c617-4d51-bf78-6ab1b776a5cd">

You can see that the style is applied only to the ngContent of the component/page.

Did the test locally by reverting the changes and leaving only one page with the change and vice-versa and it didn't propagate.

I believe this is following the approach as other components/pages.

One other way that I could implement is to create a new class such as `.a11y-link` and add this class on each link element, but IMO this would be way too similar of having inline css.

WDYT?

Reply:
Ah, thanks -- I think what you have here is fine, thanks for testing. I believe in some old pages we had an issue of leaking CSS (so what we typically do is add a prefix to the CSS rule), but if you've tested that it doesn't leak then what you have looks good. Thanks!